apps/user/src/services/notice/queries.ts

```diff
@@ -1,17 +1,18 @@
-import { noticeList } from "./api";
-import * as KEY from "@/constants/key";
-import { useQuery } from "react-query";
+import { noticeList } from './api';
+import * as KEY from '@/constants/key';
+import { useQuery } from 'react-query';
 
 // 공지사항 리스트
 interface NoticeListType {
-  id: number;
-  title: string;
-  date: string;
+    id: number;
+    title: string;
+    date: string;
 }
 
 export const useNoticeListQuery = () => {
-  const { data } = useQuery<NoticeListType[]>([KEY.NOTICE_LIST], () =>
-    noticeList()
-  );
-  return { data: data || [] };
+    return useQuery<NoticeListType[]>([KEY.NOTICE_LIST], () => noticeList(), {
+        refetchOnWindowFocus: false,
+        initialData: [],
+        staleTime: 1000 * 60,
+    });
```

Comment:
[이 PR](https://github.com/toss/slash/pull/234)을 보면서 react-query v5의 BREAKING CHANGE를 같이 준비해보면 좋을 거 같아요. [이 eslint 플러그인](https://tanstack.com/query/v4/docs/react/eslint/eslint-plugin-query)을 활용하시면 점진적인 마이그레이션이 쉬워질 거 같아요. 괜찮으시면 commit suggestion해주셔도 좋아요.

```suggestion
    return useQuery<NoticeListType[]>({
        queryKey: [KEY.NOTICE_LIST],
        queryFn: () => noticeList(),
        refetchOnWindowFocus: false,
        initialData: [],
        staleTime: 1000 * 60,
    });
```

Reply:
오 감사합니다 !! 한번 열심히 공부해서 적용해보도록하겠습니다 !!